src/styles/blocks/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  box-sizing: border-box;
```

Comment:
`box-sizing: border-box;` is duplicated, so make sense to move it to the `*{ box-sizing: border-box }`